cockatrice/src/tab_game.cpp

```diff
@@ -448,9 +451,11 @@ TabGame::~TabGame()
 void TabGame::retranslateUi()
 {
     QString tabText = " | " + (replay ? tr("Replay") : tr("Game")) + " #" + QString::number(gameInfo.game_id());
+    QString userCountInfo = QString(" %1/%2").arg(playerCountInRoom).arg(gameInfo.max_players());
+    qDebug() << "User Count Info: " << userCountInfo;
 
     cardInfoDock->setWindowTitle(tr("Card Info") + (cardInfoDock->isWindow() ? tabText : QString()));
-    playerListDock->setWindowTitle(tr("Player List") + (playerListDock->isWindow() ? tabText : QString()));
+    playerListDock->setWindowTitle(tr("Player List") + userCountInfo + (playerListDock->isWindow() ? tabText : QString()));
```

Comment:
You can get the number of players from the PlayerListWidget, just adding a method like:
```cpp
int PlayerListWidget::getPlayersCount() { return players.size(); }
```